README.md

```diff
@@ -1 +1,31 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+# Portfolio-setup-and-mobile-version-skeleton
+
+This is the mobile version skeleton of the portfolio Micronaust build during the program.
+As is the first project, we are supposed to do only the two firsts sections.
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Authors
+
+👤 **Francisco Ponce**
+
+- GitHub: [@franciscoPonceDev](https://github.com/franciscoPonceDev)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/dev-ponce/)
+
+
+Give a ⭐️ if you like this project!
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
This link also is broken. As I said the README should be written without broken links. Please, kindly fix it.